cdhweb/pages/templates/cdhpages/home_page.html

```diff
@@ -1,7 +1,7 @@
 {% extends 'base.html' %}
 {% load wagtailcore_tags %}
 
-{% block page-title %}{{ page.seo_title }}{% endblock %}
+{% block page-title %}Home{% endblock %}
```

Comment:
I don't think "Home" is actually a meaningful title, esp. out of context. I also think it would be better if this is user-editable vs hardcoded in the template

Reply:
Would like see the homepage keep the full name similar to what it has now (Center for Digital Humanities at Princeton)

Reply:
i think that's the current `seo_title`, so I'll just revert this